server/bleep/src/webserver.rs

```diff
@@ -65,7 +66,16 @@ pub async fn start(app: Application) -> anyhow::Result<()> {
         // misc
         .route("/search", get(semantic::complex_search))
         .route("/file/*ref", get(file::handle))
-        .route("/answer", answer::endpoint());
+        .route("/answer", get(answer::handle))
+        .route("/answer/conversations", get(answer::conversations::list))
+        .route(
+            "/answer/conversations/:thread_id",
+            get(answer::conversations::thread),
+        )
+        .route(
+            "/answer/conversations",
+            delete(answer::conversations::delete),
+        );
```

Comment:
the two routes for `/answer/conversations` can be merged as `.route("/answer/conversations", get(..).delete(..))`.